.github/workflows/scripts/setup_load_tests.sh

```diff
@@ -14,5 +14,5 @@ else
     curr="${TESTS[$i]}"
 fi
 done
-MATRIX+="]}"
+MATRIX+=",{\"test\":\"$curr\"}]}"
```

Comment:
This script actually misses the last two load tests. This change ensures last two load tests are added to the matrix.